drizzle/migrations/0001_sync_schema_full.sql

```
@@ -0,0 +1,80 @@
+ALTER TABLE "messages" ALTER COLUMN "role" SET DATA TYPE text;--> statement-breakpoint
+ALTER TABLE "chats" ADD COLUMN "path" text;--> statement-breakpoint
+ALTER TABLE "chats" ADD COLUMN "share_path" text;--> statement-breakpoint
+ALTER TABLE "chats" ADD COLUMN "shareable_link_id" uuid DEFAULT gen_random_uuid();--> statement-breakpoint
+ALTER TABLE "chats" ADD COLUMN "updated_at" timestamp with time zone DEFAULT now() NOT NULL;--> statement-breakpoint
+ALTER TABLE "messages" ADD COLUMN "embedding" vector(1536);--> statement-breakpoint
+ALTER TABLE "messages" ADD COLUMN "location_id" uuid;--> statement-breakpoint
+ALTER TABLE "users" ADD COLUMN "email" text;--> statement-breakpoint
+ALTER TABLE "users" ADD COLUMN "role" text DEFAULT 'viewer';--> statement-breakpoint
+ALTER TABLE "users" ADD COLUMN "selected_model" text;--> statement-breakpoint
+ALTER TABLE "users" ADD COLUMN "system_prompt" text;--> statement-breakpoint
+ALTER TABLE "calendar_notes" ADD CONSTRAINT "calendar_notes_user_id_users_id_fk" FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)